hummingbot/connector/exchange/hitbtc/hitbtc_utils.py

```diff
@@ -50,6 +50,10 @@ def generate_request_id(cls) -> int:
 
 def split_trading_pair(trading_pair: str) -> Optional[Tuple[str, str]]:
     try:
+        if trading_pair == 'USDTUSD':
+            return 'USD', 'TUSD'
+        elif trading_pair == 'USDTGUSD':
+            return 'USD', 'GUSD'
```

Comment:
```suggestion
            return 'USDT', 'GUSD'
```
This suggestion resolves the error I got locally for `USDTGUSD` pair. Kindly confirm and commit suggestion. Thanks.

Reply:
![errr](https://user-images.githubusercontent.com/31972210/126328092-6fc7abe3-3b78-4f02-b7d9-a6e107a78359.png)

Reply:
Aside that, I think we would have to add an exception to prevent translating this asset if there's no USD asset to prevent looking at the wrong asset.